value/binary.go

```diff
@@ -1172,6 +1172,32 @@ func init() {
 		},
 
 		{
+			name:      "flip",
+			whichType: atLeastVectorType,
+			fn: [numType]binaryFn{
+				vectorType: func(c Context, u, v Value) Value {
+					countVec := u.(Vector)
+					count, ok := countVec[0].(Int)
```

Comment:
What would the correct error be? I tried a couple of things on try apl.org. 
`1 2 ⊖ 3 3 ⍴ ⍳ 9` gives a length error

`1 2 3 ⊖ 3 3 ⍴ ⍳ 9` gives
`4 8 3
7 2 6
1 5 9`
I can't pretend to fully understand what is going on there.
For the moment, it seems fair to just error on anything that's not a len(1) vector.